src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccessStep2.tsx

```diff
@@ -403,12 +431,20 @@ const RequestDataAccessStep2: React.FC<RequestDataAccessStep2Props> = props => {
         accessorChanges: newAccessorChanges,
       })
     })
+
+    // Update accessor renewal radio buttons if available
+    if (isRenewal) {
+      const filteredRadio:AccessorChange[] = accessorRadioValue.filter(
+        item => item.userId !== pid
+      )
+      setAccessorRadioValue(filteredRadio)
+    }
```

Comment:
Remove the renew/revoke radio buttons if an accessor is deleted.

Reply:
This should never happen, since you should only be able to delete newly added accessors (accessors who are going to have GAIN_ACCESS in the request).  For existing accessors, there should be no option to delete - the submitter needs to pick Renew or Revoke (Revoke is the auditable delete).

Reply:
This is fixed. A user that has a renewal data access type will not have the delete "X" button.